### DISCUSSION TITLE
Refresh schedule doesn't work

### DISCUSSION THREAD

**Discussion by hanzomythus** [getredash/redash#7199] [Q&A]
Hello, could you help me?

Within my Redash application I'm trying to run "Refresh schedule" but it works in some queries and others it doesn't, I created 03 test queries but only one works with "Refresh schedule".

**Comment by justinclift** [comment]
Which version of Redash are you using?

We're currently testing some PR's that fix problems with scheduled query refreshes.  Should be finished testing soon, likely tomorrow.

**Reply by hanzomythus** [reply]
Thanks for the quick response.

I'am using version redash:10.1.0. I'll wait then!

**Reply by justinclift** [reply]
Yeah, in theory we should have something ready in a few days. :smile:

**Reply by hanzomythus** [reply]
@justinclift Hello, do you have any news about it?

**Reply by justinclift** [reply]
Yeah, we've merged one of the two PR's (#7184), and there's one more that we're discussing if it needs further modification before merging (#7111).

**Comment by Jengro777** [comment]
Refresh schedule doesn't work 

V25.1.0
